src/Development/IDE/Spans/Common.hs

```diff
@@ -80,10 +79,6 @@ spanDocToMarkdown (SpanDocString _)
 #endif
 spanDocToMarkdown (SpanDocText txt) = txt
 
-spanDocToMarkdownForTest :: String -> String
```

Comment:
this is even used in `ghcide`s own test suite, shouldn’t weeder notice this?

Reply:
Ah, I didn't build with tests. I tend to take the view that test-only code should live in test libraries, so when I run weeder I don't enable tests.